### ISSUE TITLE
[Keras] Sampling several weights per iteration

### ISSUE THREAD

**Issue by ltiao** [data61/aboleth#64]
Can be achieved with the combination of RepeatVector and TimeDistributed. Need to determine the behaviour of the loss functions, whether they are averaged or not at the end. Average of loss across samples of weights correspond to the expected log likelihood.
